connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -372,14 +379,21 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
   // checks for cancellation and throws if it's the case
   await Context.current().sleep(1);
 
-  if (pageCount > 0) {
-    await syncSucceeded(connector.id);
-  } else {
+  if (totalExtracted <= 0) {
```

Comment:
nit: style: unless there is good reason to, we'd do `=== 0` rather than `<=` (might leave other engs to wonder if value can get negative, etc.)

Reply:
also, would we put 0 here or MIN_EXTRACTED_CONTENT ? (e.g. = 1024)
This would be more robust IMO cc @albandum who has user-side context

Reply:
(the idea being that if we just extract a JS script tag with nothing else, from the user POV we didn't actually crawl so we should err)

Reply:
@albandum i'd like your take here: for pages we do not extract anything from (hydrated with js): what do you think the trigger to identify a sync error should be?

Reply:
I'd say something like 'no content on any of the pages of the site'?

But we'll still likely crawl some crap content like a footer or something  which makes detection not that easy

Reply:
hence the MIN_EXTRACTED_CONTENT